discovery_engine/lib/src/discovery_engine_base.dart

```diff
@@ -364,43 +364,44 @@ class DiscoveryEngine {
     });
   }
 
-  /// Requests a new search for [Document]s related to `queryTerm`.
+  /// Requests a new active search for [Document]s related to `queryTerm`.
   ///
   /// In response it can return:
-  /// - [SearchRequestSucceeded] for successful response, containing a list of
+  /// - [ActiveSearchRequestSucceeded] for successful response, containing a list of
   /// [Document]s
-  /// - [SearchRequestFailed] for failed response, with a reason for failure
+  /// - [ActiveSearchRequestFailed] for failed response, with a reason for failure
   /// - [EngineExceptionReason] for unexpected exception raised, with a reason
   /// for such failure.
-  Future<EngineEvent> requestSearch(String queryTerm) {
+  Future<EngineEvent> requestQuerySearch(String queryTerm) {
```

Comment:
i guess we have to inform blue about all the name changes in this public API?

Reply:
:wrench:  Given that we call it active search everywhere we probably should call it active search here too:

```suggestion
  Future<EngineEvent> requestActiveSearch(String queryTerm) {
```

Reply:
both `QuerySearch` and `TopicSearch` are subtypes of `ActiveSearch`, this is why i deviated from the naming here. we could also name these functions `requestActiveQuerySearch` and `requestActiveTopicSearch`, but i'm not sure if this is too verbose?

Reply:
I guess we could keep it that way.